packages/models/src/event.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
This looks like it's been removed upstream already, but please make sure it doesn't make it to `main` :-)